src/pages/slots/SlotConfiguration.tsx

```diff
@@ -1,69 +1,237 @@
+                        </SlotInfoContainer>
+                        <ActionsContainer>
+                            <img src={EditIcon} width={"24px"} height={"24px"}></img>
+                            <img src={DeleteIcon} width={"24px"} height={"24px"}></img>
+                        </ActionsContainer>
+                    </SpecificSlotContainer>
+                ))}
+            </SlotsContainer >
+        )
+    };
+
+    const NonExistingSlotsSkeleton = () => {
+        return (
+            <SlotsContainer>
+                <TitlesContainer>
+                    <MainTitle>Turnos del {selectSpanishValue}</MainTitle>
+                    <Subtitle>No hay turnos registrados</Subtitle>
+                </TitlesContainer>
+                <InfoContainer>
+                    <img src={CalendarIcon} width={32} height={32}></img>
+                    <SlotInfo $isBold={true}>Aún no existen turnos para este día</SlotInfo>
+                    <SlotInfo $isDown={true}>Podés registrar tu primer turno</SlotInfo>
+                </InfoContainer>
+                <Button onClick={() => { setShowModal(true); setModalType(ModalType.CREATE); }}>Crear primer turno
+                    <img
+                        src={AddIcon}
+                        width={24}
+                        height={24} />
+                </Button>
+            </SlotsContainer>
+        )
+    }
+
+    const modalConfig: Record<ModalType, ModalConfig> = {
+        [ModalType.CREATE]: {
+            contentRef: createSlotRef,
+            content: <CreateSlotForm ref={createSlotRef} />,
+            onConfirm: handleCreateSlotModal
+        },
+        [ModalType.EDIT]: {
+            contentRef: editCapacityRef,
+            content: <EditSlotForm ref={editCapacityRef} />,
+            onConfirm: handleEditCapacityModal
+        }
+    }
+
+    return (
+        <MainContainer>
+            <Title>MIS TURNOS</Title>
+            {showModal && modalType && (
+                <Modal
+                    onClose={() => setShowModal(false)}
+                    showButtons={true}
+                    contentRef={modalConfig[modalType].contentRef}
+                    onConfirm={modalConfig[modalType].onConfirm}
+                >
+                    {modalConfig[modalType].content}
+                </Modal>
+            )}
+            <SkeletonsContainer>
+                <SlotConfigurationSkeleton />
+                {selectEnglishValue === "" ? null : (
+                    (totalSlots === 0) ? (
+                        <AnimatedContainer>
+                            <NonExistingSlotsSkeleton />
+                        </AnimatedContainer>
+                    ) : (
+                        <AnimatedContainer>
+                            <VisualizeSlotsSkeleton />
+                        </AnimatedContainer>
+                    )
+                )}
```

Comment:
Esto lo podrias refactorizar asi


```suggestion
                {selectEnglishValue !== "" && (
                    <AnimatedContainer>
                        {totalSlots === 0 ? <NonExistingSlotsSkeleton /> : <VisualizeSlotsSkeleton />} 
                    </AnimatedContainer>
                )}
```